lib/web/ui/server.go

```diff
@@ -368,3 +371,24 @@ func MakeDesktopServices(windowsDesktopServices []types.WindowsDesktopService) [
 
 	return desktopServices
 }
+
+func stripProtocolAndPort(uri string) string {
```

Comment:
I think we should use https://pkg.go.dev/net/url#Parse and then `url.Host` (possibly removing the port)

Reply:
Disregard this comment
We might have the `host:port` only as `uri`.

Maybe we should change the arg name to something else then :thinking: :nail_care:

Reply:
i opted to leave a clarifying comment instead of changing arg name. from looking it up, `localhost:8080` is also a uri https://danielmiessler.com/study/difference-between-uri-url/